RecoHI/HiJetAlgos/python/HiRecoPFJets_cff.py

```diff
@@ -7,7 +7,7 @@
 #pseudo towers for noise suppression background subtraction
 PFTowers = cms.EDProducer("ParticleTowerProducer",
                           src = cms.InputTag("particleFlow"),
-                          useHF = cms.bool(False)
+                          useHF = cms.untracked.bool(False)
                           )
```

Comment:
please import from the newly created ```particleTowerProducer_cfi.py``` to minimize config replication:
```suggestion
import RecoHI.HiJetAlgos.particleTowerProducer_cfi as _mod
PFTowers = _mod.particleTowerProducer.clone(useHF = False)
```